development/compiling/compiling_for_osx.rst

```diff
@@ -35,7 +35,17 @@ Compiling
 
 Start a terminal, go to the root directory of the engine source code and type::
 
-    scons platform=osx --jobs=$(sysctl -n hw.logicalcpu)
+- for Intel (x86-64) powered Macs
```

Comment:
The previous line ends with `::` which starts a code block, I don't think this will work as expected here. I'd suggest rephrasing to whole section:
```
Start a terminal and go to the root directory of the engine source code.

To compile for Intel (x86-64) powered Macs, use::

    scons platform=osx arch=x86_64 --jobs=$(sysctl -n hw.logicalcpu)

To compile for Apple Silicon (ARM64) powered Macs, use::

    scons platform=osx arch=arm64 --jobs=$(sysctl -n hw.logicalcpu)

To support both architectures in a single "Universal 2" binary, run the above two commands and then use ``lipo`` to bundle them together::

    lipo -create bin/godot.osx.tools.x86_64 bin/godot.osx.tools.arm64 -output bin/godot.osx.tools.universal
```